jwst/jump/jump.py

```diff
@@ -105,17 +104,17 @@ def detect_jumps (input_model, gain_model, readnoise_model,
         slices.insert(i, (data[:, :, i * yincrement:(i + 1) * yincrement, :],
                           gdq[:, :, i * yincrement:(i + 1) * yincrement, :],
                           readnoise_2d[i * yincrement:(i + 1) * yincrement, :],
-                          rejection_threshold, frames_per_group, flag_4_neighbors,
+                          rejection_threshold, num_groups, flag_4_neighbors,
                           max_jump_to_flag_neighbors, min_jump_to_flag_neighbors))
     # last slice get the rest
     slices.insert(numslices - 1, (data[:, :, (numslices - 1) * yincrement:nrows, :],
                                  gdq[:, :, (numslices - 1) * yincrement:nrows, :],
                                  readnoise_2d[(numslices - 1) * yincrement:nrows, :],
-                                 rejection_threshold, frames_per_group, flag_4_neighbors,
+                                 rejection_threshold, num_groups, flag_4_neighbors,
                                  max_jump_to_flag_neighbors, min_jump_to_flag_neighbors))
     if numslices == 1:
         gdq, row_below_dq, row_above_dq = twopt.find_crs(data, gdq, readnoise_2d, rejection_threshold,
-                                                                        frames_per_group, flag_4_neighbors,
+                                                                        num_groups, flag_4_neighbors,
```

Comment:
Previously "nframes" had been passed into `find_crs`, which is in turn used within that function to properly estimate the read noise per group. Now you're sending "ngroups" for that argument, so are you still getting the correct read noise estimate? The read noise per group (or per two adjacent groups) should depend on "nframes", not "ngroups", shouldn't it?

Reply:
Well, I lost an update somewhere in the creation of this branch. You are right this should be nframes. That's the but that I introduced and fixed in the main pull request. Somehow I lost jump.py in my repo and somehow I coped in a version that had the ngroups bug.
  My git skills clearly need work. 
I'll fix it now

Reply:
I reapplied the fix to match the version of jump.py in master and then put in the small change to the default value. I also updated the test case to correctly fail when the wrong value is passed.